### PR TITLE
Saas 12.5 searchpanel aab

### PR DESCRIPTION
**Pull request by aab-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by aab-odoo** [comment]
robodoo rebase-ff

File: addons/hr/views/hr_department_views.xml [hunk 1]
```diff
@@ -75,7 +75,8 @@
                                 <div class="container o_kanban_card_content" t-if="!selection_mode">
                                     <div class="row">
                                         <div class="col-6 o_kanban_primary_left">
-                                            <button class="btn btn-primary" name="%(act_employee_from_department)d" type="action">Employees</button>
+                                            <button class="btn btn-primary" name="%(act_employee_from_department)d" type="action"
+                                                    context="{'searchpanel_default_department_id': active_id}">Employees</button>
```
Comment (LucasLefevre) [inline addons/hr/views/hr_department_views.xml]: I would rather fix this like in https://github.com/odoo/odoo/pull/36669/commits/dee7f91568a2d470aed4881f8542cb2f08dce216

Comment (aab-odoo) [reply addons/hr/views/hr_department_views.xml]: I totally agree

**Comment by VincentSchippefilt** [comment]
robodoo r+

**Pull request closed without merging**